spec.md

```diff
@@ -2387,6 +2389,58 @@ module DUT :
   connect y, p
 ```
 
+## Property Assignments
+
+Connections between property typed expressions (see [@sec:property-types]) are
+not supported in the `connect`{.firrtl} statement (see [@sec:connects]).
+
+Instead, property typed expressions are assigned with the `propassign`{.firrtl}
+statement.
+
+Property typed expressions have the normal rules for flow (see [@sec:flows]),
+but otherwise use a stricter, simpler algorithm than `connect`{.firrtl}. In
+order for a property assignment to be legal, the following conditions must hold:
+
+1. The left-hand and right-hand side expressions must be references (see
+   [@sec:references]).
```

Comment:
What does this mean?  It seems sufficient to say that th left and right hand side must be property types?

Reply:
i think he is saying you can't assign from a literal?

Reply:
I was thinking I would explicitly say the expressions where property types are allowed, but I think Schuyler is right that this is just redundant. This is already defined in the section for property types.